pkg/operator/status.go

```diff
@@ -191,3 +246,11 @@ func (r *StatusReporter) CheckMachineAPI() (bool, error) {
 	glog.Infof("machine-api-operator not ready yet")
 	return false, nil
 }
+
+func printOperandVersions(desiredVersions []osconfigv1.OperandVersion) string {
```

Comment:
nit: on `print`. it doesn't actually print, so maybe just `operandVersions()`?

Reply:
This is copied from machine-api-operator.  I'd rather keep the naming the same for that reason.